app/app.py

```diff
@@ -5,20 +5,22 @@
 
 app = Flask(__name__)
 
+CORS_HEADERS = {"Access-Control-Allow-Origin": "*"}
```

Comment:
shudders

Reply:
I have a thoughts about why this might not be an issue when we do prod build, hopefully won't be too painful